app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const {
+    getTopics
+} = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
What does `express.json()` do? Do we currently have a test in our code that needs this in order to pass?

Reply:
Not using it yet, as only working with GET. Will delete.